packages/clay-badge/src/ClayBadge.js

```diff
@@ -73,10 +30,10 @@ ClayBadge.STATE = {
 	 * The label of the badge element.
 	 * @instance
 	 * @memberof ClayBadge
-	 * @type {number}
+	 * @type {number|string}
 	 * default undefined
 	 */
-	label: Config.number().setter('hasNumberLabel_').required(),
+	label: Config.any().required(),
```

Comment:
`Config.any()` accepts anything, including functions, arrays, etc. This should be just string.